.gitignore

```diff
@@ -1,3 +1,10 @@
+	## this should ignore all files in the pipeline ##
+	
+data/input/*
+data/out/*/*.tsv
```

Comment:
This misses a number of `tsv` files that were committed (see below), since you are targeting a particular directory structure.

Instead, ignore _any_ such file by simply using `*.tsv`.